src/controllers/cart.controller.js

```diff
@@ -26,4 +27,28 @@ export class CartController {
 			});
 		}
 	}
+
+	static async viewCart(req, res) {
+		try {
+			const cart = req.cart;
+			const cartProducts = await CartService.getCartProducts(cart.products);
```

Comment:
Service to get products by ids (`getCartProducts`) should only accept an array of ids, not a cart array